### PR TITLE
Terminal, ImageWidget, and Notebook design documents

### PR DESCRIPTION
**Pull request by faricacarroll**

### CODE REVIEW

**Comment by sccolbert** [comment]
Let us know when this is ready for review.

**Comment by faricacarroll** [comment]
Documents terminal2.md, imagewidget,md. and notebook.md are now ready for review.

File: design/imagewidget.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Design of the Imagewidget
+
+This document describes the design of the imagewidget. This document illustrates how our research on personas and other
```
Comment (blink1073) [inline design/imagewidget.md]: Please use consistent capitalization throughout, e.g. ImageWidget (or image widget), Jupyter(lab), CS major.

File: design/imagewidget.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+**Goal:** To be able to manipulate images in JupyterLab to go alongside and support data analysis.
+
+Some things they would do with the imagewidget include:
+
+* Pull images from the file browser and make it larger/smaller
+* Images should be centered
+
+### College Student CS Major
+
+Leon is a sophomore cs major student who uses jupyter notebooks to share projects with his peers. He knows that JupyterLab
```
Comment (blink1073) [inline design/imagewidget.md]: Can we use a female name here since we've used a male name above?

File: design/imagewidget.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+* Resize
+* See the entire image
+
+## Other Solutions
+
+Lots of softwares use slightly different ways to resize images. When you insert an image into Illustrator, you can either double click
+on the artboard and the image will be inserted at its normal file size, or you can click and drag a square for the image size.
+
+In google slides, you click to insert an image and the image is placed in its actual size onto the slide. A surrounding highlight
```
Comment (blink1073) [inline design/imagewidget.md]: Google*

**Comment by blink1073** [comment]
@willingc, do we have a style guide for Markdown?  If not, we should start one with things like word wrap and headings.

File: design/notebook.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+* Import data into a pandas DataFrame
+* Convert measurements in the data with programming
+* Cleaning up the data
+* Program the mean imputation
+* Save the data file
+* Save a copy
+
+### Student using the notebook in a programming class
+
+The student is a first-year cs major who is taking a course where the professor wants his students to use JupyterLab to write
```
Comment (blink1073) [inline design/notebook.md]: Again, please use men and women equally in examples.

File: design/notebook.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+The experienced data scientist is familiar with the Jupyter Notebook and uses it regularly. He has become accustomed to the shortcuts
+and toolbars. He uses the notebook to take measurements from plants and identify the species based on those measurements alone.
+
+**Goal:** Use the Notebook to display his various visualizations for his data
+
+Some things they would in the notebook include:
+* Create a scatterplot matrix
+* Import data into a pandas DataFrame
+* Convert measurements in the data with programming
+* Cleaning up the data
```
Comment (blink1073) [inline design/notebook.md]: Clean up the data*

**Comment by willingc** [comment]
@blink1073 You rock! Great suggestion on alternating gender and a style guide. Opened an issue on jupyter/jupyter. ☀️ 

@faricacarroll I'll be in this morning if you need any help re: markdown and review changes 🌻

**Comment by faricacarroll** [comment]
@blink1073 Thanks for all the great suggestions! I've implemented the alternating genders for the personas. Great point! 👍 

@willingc Thank you! 😸 

Brian went over our design documents yesterday so we've made a lot of changes since then. The user tasks section is much more in depth now.

File: design/imagewidget.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Design of the imagewidget plugin
+This document describes the design of the imagewidget plugin. This document illustrates how our research on personas and other solutions translates into the design decisions made to improve the imagewidget plugin.
+
+# Personas
+### Jackie Lair
+Jackie Lair is an experienced data scientist who is familiar with JupyterLab and uses it regularly. He uses the imagewidget to compare MRI scans alongside her code and write programs that outputs the information found in the MRI scans.
```
Comment (blink1073) [inline design/imagewidget.md]: Should be "She uses".

File: design/launcher.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# Design of the launcher plugin
+This document describes the design of the launcher plugin. This document illustrates how our research on personas and other solutions translates into the design decisions made to improve the launcher plugin.
+
+# Personas
+### William Shar
```
Comment (blink1073) [inline design/launcher.md]: William is typically a male name, but who are we to judge ;).

File: design/terminal.md [hunk 8]
```diff
@@ -1,54 +1,38 @@
 # Design of the terminal plugin
-
-This document describes the design of the terminal plugin. Any significant change
-to the terminal plugin should also involve changes here.
+This document describes the design of the terminal plugin. Any significant change to the terminal plugin should also involve changes here.
 
 # Personas
+### Janet Tobin 
+An experienced data scientist with a graduate degree in a technical field and has been coding for more than 5 years in multiple languages. She is 35 years old, has a solid income, and lives in an urban setting. In the past she has used the Jupyter Notebook alongside a text editor (Sublime Text) and the Terminal app on a Mac. She lovse to code and feels at home in a terminal.
```
Comment (blink1073) [inline design/terminal.md]: sp (lovse)

**Comment by blink1073** [comment]
Looks good to me, let's merge and iterate.  Thanks @faricacarroll and team!

**Pull request merged**